src/epic.rs

```diff
@@ -302,6 +302,665 @@ pub static EPICS: &[Epic] = &[
         title: "Yadegar-e Zariran",
         wiki: Wiki::Same,
     },
+    // 16th century
+    Epic {
+        title: "Lilit Phra Lo",
+        wiki: Wiki::Name("Phra Lo"),
+    },
+    Epic {
+        title: "Judita",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Orlando Furioso",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Davidiad",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Christiad",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Os Lusíadas",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "L'Amadigi",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "La Araucana",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "La Gerusalemme liberata",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Ramacharitamanasa",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Journey to the West",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "The Faerie Queene",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Venus and Adonis",
+        wiki: Wiki::Name("Venus and Adonis (Shakespeare poem)"),
+    },
+    Epic {
+        title: "Lucrece",
+        wiki: Wiki::Name("The Rape of Lucrece"),
+    },
+    // 17th century
+    Epic {
+        title: "La Argentina",
+        wiki: Wiki::Name("La Argentina (poem)"),
+    },
+    Epic {
+        title: "La Cleopatra",
+        wiki: Wiki::Name("La Cleopatra (poem)"),
+    },
+    Epic {
+        title: "Biag ni Lam-ang",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Il Conquisto di Granata",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Exact Epitome of the Four Monarchies",
+        wiki: Wiki::Name("The Tenth Muse Lately Sprung Up in America"),
+    },
+    Epic {
+        title: "Szigeti veszedelem",
+        wiki: Wiki::Name("Peril of Sziget"),
+    },
+    Epic {
+        title: "Gondibert",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Paradise Lost",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Paradise Regained",
+        wiki: Wiki::Same,
+    },
+    Epic {
+        title: "Khun Chang Khun Phaen",
```

Comment:
This appears to be a duplicate of line 913. https://github.com/a2aaron/mario-epics/blob/75fe16cfe64105f8636c35de92a1d1d76a398b81/src/epic.rs#L912-L915